can-realtime-rest-model.md

```diff
@@ -314,7 +314,7 @@ Todo.connection = realtimeRestModel({
 
 class TodoDetails extends StacheElement {
     static view = `
-        Todo Name is: {{todoPromise.value.name}}
+        Todo name is: {{this.todoPromise.value.name}}
```

Comment:
can you update the spacing on this one while we're here? `{{ this.todoPromise.value.name }}`.